src/app/containers/Navigation.js

```diff
@@ -20,7 +20,7 @@ class Navigation extends Component {
   }
 
   renderTrialLinks() {
-    if (!trialMode) return null
+    if (!trialMode || process.env.NODE_ENV == 'development') return null
```

Comment:
```suggestion
    if (!trialMode || isDev) return null
```
There's also this const on line 15 ready to go, if it doesn't come back to bite anywhere else. Looks good either way